src/main/java/org/seqdoop/hadoop_bam/util/FileFormatUtils.java

```diff
@@ -0,0 +1,59 @@
+package org.seqdoop.hadoop_bam.util;
+
+import com.google.common.collect.ImmutableList;
+import htsjdk.samtools.util.Interval;
+import org.apache.hadoop.conf.Configuration;
+import org.seqdoop.hadoop_bam.FormatException;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Supplier;
+
+/**
+ * Common utilities across different file formats.
+ */
+public abstract class FileFormatUtils {
```

Comment:
Rather than make this class abstract, it would be more idiomatic to add a private constructor so it can't be instantiated.

Reply:
Sure.